doc/helm-values.adoc

```diff
@@ -356,6 +356,18 @@ Valid values:: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and
 Description:: Tolerations to pass to the controller pods.
 
 
+=== `operator.controller.additonalEnv`
+Default:: `[]`
+Valid values:: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/[EnvVar list]
+Description:: A list of additional environment variables to pass to the Linstor controller container.
+
+
+=== `operator.controller.additionalProperties`
+Default:: `[]`
+Valid values:: list of `{ key: k, value: v }` pairs
```

Comment:
Why such pairs and not a yaml map? Is there precedent somewhere?

Reply:
I was following kubernetes API conventions: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#lists-of-named-subobjects-preferred-over-maps

Reading it again, there is an explicit exception for "pure" maps, so I think we could use a map here, too. I don't really care, I've just gotten used to the kubernetes "everything is a list" approach.

Reply:
I would choose a map in this case, since, as you say, it is a "pure" map. Might we ever add additional keys to the objects? Actually, maybe we might want to treat aux properties differently? In that case, we should add that distinction now.

It looks like `/` can appear unquoted in a yaml key, so all the keys should be easily usable in a yaml map.